src/components/Bookings.js

```diff
@@ -0,0 +1,73 @@
+import React, { useEffect, useState } from "react";
+import Search from "./Search.js";
+import SearchResults from "./SearchResults";
+import CustomerProfile from "./CustomerProfile.js";
+
+const Bookings = () => {
+  const [bookings, setBookings] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(false);
+  const [customerProfile, setCustomerProfile] = useState([]);
+
+  useEffect(() => {
+    const getBookings = async () => {
+      const res = await fetch("https://cyf-react.glitch.me");
+
+      if (res.status === 200) {
+        const data = await res.json();
+        setBookings(data);
+
+        setLoading(false);
+      } else {
+        setLoading(false);
+        setError(true);
+      }
+    };
+
+    getBookings();
+  }, []);
+
+  const search = searchVal => {
```

Comment:
I would rename this function - make it's name a bit more specific. What exactly is this function searching for? This might help you to come up with a more specific name.